code/modules/antagonists/wizard/equipment/wizard_spellbook.dm

```diff
@@ -0,0 +1,329 @@
+/obj/item/spellbook
+	name = "spell book"
+	desc = "An unearthly tome that glows with power."
+	icon = 'icons/obj/library.dmi'
+	icon_state ="book"
+	worn_icon_state = "book"
+	throw_speed = 2
+	throw_range = 5
+	w_class = WEIGHT_CLASS_TINY
+
+	/// The number of book charges we have to buy spells
+	var/uses = 10
+	/// The bonus that you get from going semi-random.
+	var/semi_random_bonus = 2
+	/// The bonus that you get from going full random.
+	var/full_random_bonus = 5
```

Comment:
These two got completely removed in the spellbook tgui pr because wild magic was still a thing there, since wild magic gets removed in this pr it would also need to be added back to the new tgui file

Reply:
Are you sure the random thing got removed
  {
    title: 'Randomize',
    blurb: "If you didn't like the loadouts offered, you can embrace chaos. Not recommended for newer wizards.",
    component: () => <Randomize />,
  },

from your .js

I can remove those if you want, But doesn't seem like removed to me